updater/Gemfile.lock

```diff
@@ -330,4 +330,4 @@ DEPENDENCIES
   webmock (~> 3.18)
 
 BUNDLED WITH
-   2.2.20
+   2.3.22
```

Comment:
I think once the other issues are fixed, this becomes the only line that actually changes in this lockfile under the new Bundler version...

Reply:
Indeed, after rebasing on top of #5972 this is the only change left